bundle/tests/include_test.go

```diff
@@ -31,7 +30,6 @@ func TestIncludeWithGlob(t *testing.T) {
 
 	job := b.Config.Resources.Jobs["my_job"]
 	assert.Equal(t, "1", job.ID)
-	assert.Equal(t, "include_with_glob/job.yml", filepath.ToSlash(job.ConfigFilePath))
```

Comment:
same

Reply:
Done. I'm skipping this for the other unit tests, though, since it's really functionality from the dyn packages. Let me know if you disagree and would like to retain coverage.

Reply:
+1, seems like this is no longer related. Still good to keep coverage for it in a test that isn't specific to a resource.